data/data-pipeline/data_pipeline/tests/sources/national_risk_index/data/input.csv

```diff
@@ -1,6 +1,3 @@

```

Comment:
Do you mind if we do like 10-20 rows, for robustness?

Reply:
Sure thing